instfiles/pam.d/mkpamrules

```diff
@@ -8,7 +8,7 @@ outfile="$3"
 
 service="xrdp-sesman"
 pamdir="/etc/pam.d"
-pamdir_suse="/usr/etc/pam.d"
+pamdir_suse="/usr/lib/pam.d"
```

Comment:
Since this is a shell script, how about:-

```bash
pamdir_suse="/usr/lib/pam.d"
if [ ! -d "$pamdir_suse" ]; then
    # Pre-Tumbleweed setting
    pamdir_suse="/usr/etc/pam.d"
fi
```

Reply:
Done, thanks for the suggestion!